src/Interfaces/AbstractPolygon.jl

```diff
@@ -140,6 +140,10 @@ directions using the right-hand rule (see [`is_right_turn`](@ref)).
 In particular, this method does not use the arctangent.
 """
 function <=(u::AbstractVector{N}, v::AbstractVector{N}) where {N<:Real}
+    return _leq_quadrant(u, v)
```

Comment:
I think we should `assert` that the vectors have length 2 here (I know we did not do this before). `<=` may easily be used in wrong places.